src/runtime/HalideRuntimeOpenGLCompute.h

```diff
@@ -0,0 +1,50 @@
+#ifndef HALIDE_HALIDERUNTIMEOPENGLCOMPUTE_H
+#define HALIDE_HALIDERUNTIMEOPENGLCOMPUTE_H
+
+#include "HalideRuntime.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** \file
+ *  Routines specific to the Halide OpenGL Compute runtime.
+ */
+
+extern const struct halide_device_interface *halide_openglcompute_device_interface();
+
+/** These are forward declared here to allow clients to override the
+ *  Halide Glsl runtime. Do not call them. */
+// @{
+extern int halide_openglcompute_initialize_kernels(void *user_context, void **state_ptr,
+                                            const char *src, int size);
+
+extern int halide_openglcompute_run(void *user_context,
+                             void *state_ptr,
+                             const char *entry_name,
+                             int blocksX, int blocksY, int blocksZ,
+                             int threadsX, int threadsY, int threadsZ,
+                             int shared_mem_bytes,
+                             size_t arg_sizes[],
+                             void *args[],
+                             int8_t is_buffer[],
+                             int num_attributes,
+                             float* vertex_buffer,
+                             int num_coords_dim0,
+                             int num_coords_dim1);
+// @}
+
+/** This functions MUST be provided by the host environment to retrieve pointers
```

Comment:
Is this comment really true? LLVM_Runtime_Linker seems to indicate that there's a working default implementation for many platforms.